addons/account/models/account_config_settings.py

```diff
@@ -25,6 +25,10 @@ class AccountConfigSettings(models.TransientModel):
     chart_template_id = fields.Many2one('account.chart.template', string='Template',
         domain="[('visible','=', True)]")
     code_digits = fields.Integer(string='# of Digits *', related='company_id.accounts_code_digits', help="No. of digits to use for account code")
+    sale_tax_id = fields.Many2one('account.tax', string="Default Sale Tax",
+        related='company_id.accounts_sale_tax_id', oldname="default_sale_tax")
```

Comment:
If it's related non stored you don't need to put oldname (even more, maybe this will be a problem...)